src/GeneratedMapper/Information/AfterMapInformation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using GeneratedMapper.Parsers;
+using Microsoft.CodeAnalysis;
+
+namespace GeneratedMapper.Information
+{
+    internal sealed class AfterMapInformation
+    {
+        public IMethodSymbol MethodSymbol { get; }
+
+        public AfterMapInformation(IMethodSymbol methodSymbol, ParameterParser parameterParser)
+        {
+            MethodSymbol = methodSymbol;
+            Parameters = parameterParser.ParseMethodParameters(methodSymbol.Parameters);
+        }
+
+        public ITypeSymbol PartOfType => MethodSymbol.ContainingType;
+        public string MethodName => MethodSymbol.Name;
+        public IEnumerable<ITypeSymbol?> ParameterTypes => MethodSymbol.Parameters.Select(_ => _.Type);
```

Comment:
I seen that you use `_` for these lambda's. I tend to use `_` only for when I discard a parameter, and use `x` for when I use the parameter.

Reply:
I can change that back to x.  I guess I just missed that when working on this. Also I notice there's x lambdas inside x lambdas but the compiler now knows how to read each one correctly.  For those I usually changed the sub lambda to parameter to match type name in those instances.